pycfmodel/model/resources/properties/policy_document.py

```diff
@@ -241,14 +241,14 @@ def get_iam_actions(self, difference=False):
                 if wildcard > 0:
                     actions = [
                         iam for iam in self.IAM_ACTIONS
-                        if iam.startswith(action[:wildcard])
+                        if action[:wildcard].lower() in iam.lower()
                     ]
                 elif wildcard == 0:
                     actions = self.IAM_ACTIONS
-                elif action in self.IAM_ACTIONS:
-                    actions = [action]
+                elif action.lower() in [iam.lower() for iam in self.IAM_ACTIONS]:
```

Comment:
This section addresses actions being overwritten on each iteration

Reply:
Ohh nice catch!